akka-testkit/src/main/scala/akka/testkit/TestKit.scala

```diff
@@ -688,6 +725,17 @@ trait TestKitBase {
     TestKit.shutdownActorSystem(actorSystem, duration, verifySystemShutdown)
   }
 
+  /**
+   * Spawns an actor as a child of this test actor, and returns the child's ActorRef.
+   * @param props Props to create the child actor
+   * @param name Actor name for the child actor
+   * @param supervisorStrategy Strategy should decide what to do with failures in the actor.
+   */
+  def childActorOf(props: Props, name: Option[String] = None, supervisorStrategy: Option[SupervisorStrategy] = None): ActorRef = {
+    testActor ! Spawn(props, name, supervisorStrategy)
+    expectMsgType[ActorRef]
+  }
+
```

Comment:
Mmm I thought I created methods with several arguments for the Java API. Might be this particular test code is using the scala `TestProbe` instead of the java one? I'll have a look.